### PR TITLE
Rename back "tag-pre-release-weight" and "tag-prefix"

### PR DESCRIPTION
**Pull request by ni-jsuchocki**
## Description
Renamed
label-pre-release-weight => tag-pre-release-weight
label-prefix => tag-prefix

## Related Issue
https://github.com/GitTools/GitVersion/issues/3480

### CODE REVIEW

**Comment by arturcic** [comment]
@ni-jsuchocki please run these scripts `./build.ps1 -Stage build -Target BuildPrepare `then the `./build.ps1 -Stage docs -Target GenerateSchemas` to update the schemas

**Comment by ni-jsuchocki** [comment]
Hmm I'm on Mac and I'm having hard time making the script work (managing various versions of dotnet on Mac is a mess). I have powershell installed and I ran the first command successfully, but when I run `GenerateSchemas` it tries to update a 5.11 schema for some reason (which doesn't exist in the repo):
```
Schema version: 5.11
Writing schema to .../GitVersion/schemas/5.11/GitVersion.configuration.json
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '.../GitVersion/schemas/5.11/GitVersion.configuration.json'.
```

Any idea what causes this? The gitversion binary I have installed is 6.0.0:
```
gitversion /version
6.0.0-alpha.1+Branch.main.Sha.fc5b3ff045ea0e0f59c9bf27096af9094ca17f0e
```

**Comment by arturcic** [comment]
`GenerateSchemas` gets its version by running the current gitversion from source code and not using the gitversion tool you have installed. For some reason running the gitversion against the gitversion source code you get 5.11 as the version. probably your fork does not have the tags for 5.12.0 and for 6.0.0 betas> Maybe you need to fetch them from the GitTools/GitVersion origin

**Comment by arturcic** [comment]
I would say squash them

**Comment by arturcic** [comment]
@ni-jsuchocki can you please rebase your changes on `main` ?

File: src/GitVersion.Core.Tests/IntegrationTests/VersionInTagScenarios.cs [hunk 1]
```diff
@@ -24,12 +24,12 @@ public void TagPreReleaseWeightIsNotConfigured_HeadIsATaggedCommit_WeightedPreRe
     }
 
     [Test]
-    public void LabelPreReleaseWeightIsConfigured_HeadIsATaggedCommit_WeightedPreReleaseNumberShouldBeTheSameAsTheLabelPreReleaseWeight()
+    public void TagPreReleaseWeightIsConfigured_HeadIsATaggedCommit_WeightedPreReleaseNumberShouldBeTheSameAsTheLabelPreReleaseWeight()
```
Comment (arturcic) [inline src/GitVersion.Core.Tests/IntegrationTests/VersionInTagScenarios.cs:27]:
```suggestion
    public void TagPreReleaseWeightIsConfigured_HeadIsATaggedCommit_WeightedPreReleaseNumberShouldBeTheSameAsTheTagPreReleaseWeight()
```

**Comment by arturcic** [comment]
@ni-jsuchocki can you check the comments?

**Comment by ni-jsuchocki** [comment]
Sure sure, noticed that right after I rebased.

**Review by arturcic — APPROVED** [review]

**Pull request merged**